.github/workflows/tests.yaml

```diff
@@ -41,5 +41,5 @@ jobs:
       # https://github.nrel.gov/MBAP/fastsim/issues/310
       - name: Python unit tests 
         run: |
-          pip install -e ".[dev]" && pytest -v python/fastsim/tests/ 
```

Comment:
looks like maybe we should undo these changes in a new commit, change the PR description from `attempt to fix -e in actions` to `failed attempt to fix -e in actions with other minor patch changes